Remove the need for subjects.json (#66) (#89)
metakgp/gyft: README.md

        For google calendar, follow the instructions given [here](https://support.google.com/calendar/answer/37118?hl=en).

## Development Notes 

> - This project uses the [iitkgp-erp-login](https://github.com/proffapt/iitkgp-erp-login-pypi/) package for ERP login

Reviewer: ```suggestion
This project utilizes the [iitkgp-erp-login](https://github.com/proffapt/iitkgp-erp-login-pypi/) package to handle ERP login functionality.
```
